Script/SceneSwitcher.gd

```diff
@@ -0,0 +1,55 @@
+extends Node
+
+const MAIN_MENU_SCENE := preload("res://Scene/WorldSelector.tscn") as PackedScene
+var _world_rx := RegEx.create_from_string("^res://(?<world>.+)/World.tscn$")
+
+## Proxy object for the 'window' DOM object, or null if not running on the web
+var _window: JavaScriptObject
+
+
+func _ready() -> void:
+	if OS.has_feature("web"):
+		_window = JavaScriptBridge.get_interface("window")
+		_restore_from_hash.call_deferred()
+
+
+## On the web, load the world indicated by the URL hash, if any.
+func _restore_from_hash():
+	var hash := _window.location.hash as String
+	if hash:
+		var world := hash.right(-1).uri_decode()
+		var scene := "res://" + world + "/World.tscn"
```

Comment:
There is [`String.path_join()`](https://docs.godotengine.org/en/latest/classes/class_string.html#class-string-method-path-join) but I don't think it's worth in GDScript, as unlike Python all OSs are having the same `/` path separation.

Reply:
I switched this to use path_join – I used that elsewhere in this file but forgot it existed here. Much of a muchness though!